test/docker/test_readiness.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Download jq
+curl http://stedolan.github.io/jq/download/linux64/jq -o ~/bin_compciv/jq
+chmod a+x ~/bin_compciv/jq
+
+# Define readiness function
+check_readiness() {
+   local response=$(curl -s "${GITLAB_URL}/-/readiness")
+
+   local db_check=$(echo "$response" | jq '.db_check.status')
+   local redis_check=$(echo "$response" | jq '.redis_check.status')
+   local cache_check=$(echo "$response" | jq '.cache_check.status')
+   local queues_check=$(echo "$response" | jq '.queues_check.status')
+   local shared_state_check=$(echo "$response" | jq '.shared_state_check.status')
+   local gitaly_check=$(echo "$response" | jq '.gitaly_check.status')
+
+   return db_check == "ok" && redis_check == "ok"  && cache_check == "ok" && queues_check == "ok"  && shared_state_check == "ok" && gitaly_check == "ok" 
```

Comment:
You still need to prefix these guys 🗡 
```suggestion
   return $db_check == "ok" && $redis_check == "ok"  && $cache_check == "ok" && $queues_check == "ok"  && $shared_state_check == "ok" && $gitaly_check == "ok" 
```

Reply:
Most likely you will need to write something like this

```suggestion
   if [ "$db_check" == "ok" && "$redis_check" == "ok" && "$cache_check" == "ok" && "$queues_check' == "ok" && "$shared_state_check" == "ok" && "$gitaly_check" == "ok"  ]; then
     true
   else
     false
   fi
```